src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2633,6 +2633,13 @@ RAG\ minimum\ score\ must\ be\ a\ number=RAG minimum score must be a number
 RAG\ minimum\ score\ must\ be\ greater\ than\ 0\ and\ less\ than\ 1=RAG minimum score must be greater than 0 and less than 1
 Temperature\ must\ be\ a\ number=Temperature must be a number
 If\ you\ have\ chosen\ %0\ as\ an\ AI\ provider,\ the\ privacy\ policy\ of\ %0\ applies.\ You\ find\ it\ at\ %1.=If you have chosen %0 as an AI provider, the privacy policy of %0 applies. You find it at %1.
+Generate\ embeddings\ for\ linked\ files\ in\ the\ group=Generate embeddings for linked files in the group
+Generate\ summaries\ for\ entries\ in\ the\ group=Generate summaries for entries in the group
+Generating\ summaries\ for\ %0=Generating summaries for %0
+Ingestion\ started\ for\ group\ "%0".=Ingestion started for group "%0".
+Summarization\ started\ for\ group\ "%0".=Summarization started for group "%0".
+Unable\ to\ generate\ embeddings.\ No\ library\ is\ selected.=Unable to generate embeddings. No library is selected.
+Unable\ to\ generate\ summaries.\ No\ library\ is\ selected.=Unable to generate summaries. No library is selected.
```

Comment:
I think, these can be removed (see above) --> logger.warn outputs are not localized.

Reply:
See https://github.com/JabRef/jabref/pull/11832#discussion_r1785753759